### PR TITLE
Added C# samples to "making plugins" doc

### PR DESCRIPTION
**Pull request by juspky**
I added C# samples to https://docs.godotengine.org/en/3.1/tutorials/plugins/editor/making_plugins.html

I have not found any tool that supported previewing.
```
.. tabs::
 .. code-tab:: gdscript GDScript
   code
 .. code-tab:: csharp
   more code
```
So I just hope everything is formatted correctly

### CODE REVIEW

**Comment by mhilbrunner** [comment]
Welcome and thank you!

If your PR concerns Godot in general and not only a specific version like 3.1, it should go against the **master** branch, not a branch like **3.1**.

As for the previewing, the easiest (only?) way is to let the docs build locally so you can have a look, see the README on this: https://github.com/godotengine/godot-docs/blob/master/README.md#building-with-sphinx

File: tutorials/plugins/editor/making_plugins.rst [hunk 1]
```diff
@@ -82,6 +96,27 @@ structure:
...
         pass
 
+ .. code-tab:: csharp
+ 
+    #if TOOLS
+    using Godot;
+    using System;
+
+    [Tool]
+    public class NoHeroTools : EditorPlugin
```
Comment (neikeq) [inline tutorials/plugins/editor/making_plugins.rst]: Shouldn't this be `CustomNode`?

Comment (juspky) [reply tutorials/plugins/editor/making_plugins.rst]: I guess the copy and paste cat sneaked in

**Comment by juspky** [comment]
Ugh.
I don't know how to make change this to merge into master without screwing this PR up.

**Comment by mhilbrunner** [comment]
Don't worry, I'll merge it manually, just keep it in mind for future PRs - thanks for your work :)

**Comment by mhilbrunner** [comment]
I build this locally and it looks good to go. Again, thank you!

**Comment by mhilbrunner** [comment]
Cherrypicked into master as https://github.com/godotengine/godot-docs/commit/323b13df722eb9ac2faae83951e486fece3ec28b

**Pull request merged**